index.html

```diff
@@ -20,17 +20,29 @@
       referrerpolicy="no-referrer"
     />
     <link rel="icon" href="./assets/coding.png" type="image/x-icon" />
-  </head>sectionbody>
+  </head>
     <header>
       <a id="logo" href="#">Welcome</a>
-    
-      <div class="nav-desktop"></div>
+<ul class="nav-desktop">
+  <li>Portfolio</li>
+  <li>About</li>
+  <li>Contact</li>
+  
+</ul>
+  
 
       <div class="nav-mobile">
         <nav class="btn"><i class="fa-solid fa-bars"></i></nav>
       </div>
     </header>
 
+    
+
+
+
+
+
+    
     <section class="welcome-section">
       <div class="welcome-header-text">
         <h1>Hey there. I'm Kano</h1>
```

Comment:
- [ ] Please add the right font size and weight for navigation links above.
- [ ] please add the proper margin value between text elements inside this section.
- [ ] Kindly add the proper margin value between social icons images.
- [ ] please fix the background positioning or text width because you can see the text is overlapping with the background image.
All these changes to match the given design and try to stick as much as possible to the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/159785772-c0a829d2-a494-4e76-a523-9da0139619c7.png)  | ![image](https://user-images.githubusercontent.com/68796232/159785851-58d2e074-3e35-4fc2-9095-86a357efccd8.png) |

- [ ] and also please fix it in the mobile version.

![image](https://user-images.githubusercontent.com/68796232/159788210-11dcf62f-2ba9-4eeb-b545-8ece12489259.png)